### PR TITLE
a new package: papyrus.

### PR DESCRIPTION
**Pull request by jungwonkim**
Papyrus is a distributed key-value store developed from the ECP PROTEAS project.

### CODE REVIEW

**Comment by jungwonkim** [comment]
Can I solve this travis-ci issue?
When I tried "spack test test_compiler_cmd" using python 2.7.13 on my laptop, it passed.

**Comment by adamjstewart** [comment]
I restarted the tests. I think this is just a transient bug.

**Comment by jungwonkim** [comment]
I've changed the md5 checksum

File: var/spack/repos/builtin/packages/papyrus/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+##############################################################################
+from spack import *
+
+
+class Papyrus(CMakePackage):
+    """Parallel Aggregate Persistent Storage"""
+
+    homepage = "https://code.ornl.gov/eck/papyrus"
+    url      = "https://code.ornl.gov/eck/papyrus" \
+               "/repository/archive.tar.bz2?ref=v1.0.0"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/papyrus/package.py]: You don't need to break this line, we ignore URLs when checking line lengths.

Comment (jungwonkim) [reply var/spack/repos/builtin/packages/papyrus/package.py]: Ok, thank you for letting me know.

**Pull request merged**